financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsSessionManifest.kt

```diff
@@ -133,12 +132,8 @@ internal data class FinancialConnectionsSessionManifest(
     @SerialName(value = "connected_account_name")
     val connectedAccountName: String? = null,
 
-    // @SerialName(value = "experiment_assignments")
-    @Transient
-    // TODO@carlosmuvi revert hardcoded assignments.
-    val experimentAssignments: Map<String, String>? = mapOf(
-        "native" to "true"
-    ),
+    @SerialName(value = "experiment_assignments")
+    val experimentAssignments: Map<String, String>? = null,
```

Comment:
Had this hardcoded so that I could keep testing native, are we deploying the backend for this soon? otherwise we can revert these changes once you've tested this 😆